src.ts/tests/ffi.spec.ts

```diff
@@ -1,12 +1,14 @@
 import { bytesToHex } from '@ethereumjs/util'
 import { beforeAll, describe, expect, test } from 'vitest'
 
-import { VerkleFFI, getTreeKey, getTreeKeyHash } from '../index.js'
+import { initVerkleWasm, getTreeKey, getTreeKeyHash, Context } from '../index.js'
 
 describe('bindings', () => {
-  let ffi: VerkleFFI
-  beforeAll(() => {
-    ffi = new VerkleFFI()
+  let ffi: Context;
+  let verkle;
```

Comment:
Now that this initVerkleWasm() returns void, we can probably not assign it right?